src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -3254,6 +3255,43 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         return _terminal->GetTaskbarProgress();
     }
 
+    winrt::fire_and_forget TermControl::InvertScreenColors()
+    {
+        if (!_invertTimer)
+        {
+            auto weakThis{ get_weak() };
+            co_await winrt::resume_foreground(Dispatcher());
+
+            if (auto control{ weakThis.get() })
+            {
+                auto lock = control->_terminal->LockForWriting();
+                DispatcherTimer invertTimer;
+                invertTimer.Interval(std::chrono::milliseconds(2000));
+                invertTimer.Tick({ get_weak(), &TermControl::_InvertTimerTick });

```

Comment:
What happens if you try to flash the taskbar again within the 2 second interval?

Reply:
What happens if output appears during the 2 second interval? (like, you have commands executing back to back).

Reply:
> What happens if you try to flash the taskbar again within the 2 second interval?

Nothing - subsequent attempts to flash the window while the window is already 'flashing' are ignored (that's why we check whether the timer exists before deciding to create a new one)

> What happens if output appears during the 2 second interval? (like, you have commands executing back to back).

Output happens as normal